src/confluent_kafka/src/Admin.c

```diff
@@ -4523,6 +4621,32 @@ static PyObject *Admin_c_DeletedRecords_to_py (const rd_kafka_topic_partition_li
         return NULL;
 }
 
+static PyObject *Admin_c_ElectLeadersResult_to_py(
+    const rd_kafka_topic_partition_result_t **partitions,
+    size_t cnt) {
+        PyObject *result = NULL;
+        size_t i;
+
+        result = PyDict_New();
+
+        for (i = 0; i < cnt; i++) {
+                PyObject *value = NULL;
+                const rd_kafka_topic_partition_t *rktpar;
+                const rd_kafka_error_t *error;
+
+                rktpar =
+                    rd_kafka_topic_partition_result_partition(partitions[i]);
+                error = rd_kafka_topic_partition_result_error(partitions[i]);
+                value = KafkaException_new_or_none(rd_kafka_error_code(error),
+                                                   rd_kafka_error_string(error));
+               
+                PyDict_SetItem(result, c_part_to_py(rktpar), value);
```

Comment:
Create new variable for `c_part_to_py(rktpar)`. Decrease reference for the created variable. Check other places.